pages/student/02-am_i_good_enough.md

```diff
@@ -6,13 +6,13 @@ permalink: student/am-i-good-enough.html
 sidebar: student_sidebar
 ---
 
-Do you have *some* programming experience at the university level? Then, yes, you are good enough! No, you don't need to be a Computer Science or IT major. Students from all subject areas are successful GSoC students. Have work experience programming but spend your time studying philosophy full-time? Yes, you are good enough to be a GSoC student!
+Do you have *some* programming experience and are comfortable in at least one or two programming langauges? Then, yes, you are good enough! No, you don't need to be a Computer Science or IT major or have been coding for 5 years. Have work experience programming but spend your time studying philosophy? Yes, you are good enough to be a GSoC contributor!
```

Comment:
I'd take out the 5 years bit.  GSoC is not meant for beginner programmers, but beginner open source people.  Trying to quantify either of those things is complex, so easiest to avoid it.